slick/src/main/scala/slick/jdbc/JdbcActionComponent.scala

```diff
@@ -208,17 +203,27 @@ trait JdbcActionComponent extends SqlActionComponent { self: JdbcProfile =>
         case ParameterSwitch(cases, default) =>
           findSql(cases.find { case (f, n) => f(param) }.map(_._2).getOrElse(default))
       }
-      (tree match {
+      val (comment, tree2) = tree match {
```

Comment:
`tree2` seems like a vague name for the variable.

Reply:
Updated to newTree